packages/gui/src/components/nfts/gallery/NFTGallery.tsx

```diff
@@ -65,6 +65,8 @@ export default function NFTGallery() {
     defaultCacheSizeLimit,
   );
 
+  const [sensitiveContentObject] = useLocalStorage('sensitive-content');
```

Comment:
Needs a default, otherwise sensitiveContentObject is undefined and leads to an exception on line 95.

```suggestion
  const [sensitiveContentObject] = useLocalStorage('sensitive-content', {});
```